skins/cat17/address-form/src/types.ts

```diff
@@ -1,6 +1,18 @@
+export const enum Validity {
+    INVALID,
+    VALID,
+    INCOMPLETE
```

Comment:
As far as I understand that means INCOMPLETE is assigned with the value of 2. I think this is kind of strange but it does work for our use case.
Before, we had this mean `null` which is evaluated to `false`, while `2` is `true`. But again, it works, so no objections.